docsite/rst/developing_plugins.rst

```diff
@@ -56,12 +72,20 @@ If you want more Jinja2 filters available in a Jinja2 template (filters like to_
 
 Jump into `lib/ansible/plugins/filter <https://github.com/ansible/ansible/tree/devel/lib/ansible/plugins/filter>`_ for details.
 
+Test Plugins
+------------
+
+If you want more Jinja2 tests available in a Jinja2 template (tests like 'even' and 'odd' are provided by default), they can be extended by writing a test plugin.  Most of the time, when someone comes up with an idea for a new test they would like to make available in a playbook, we'll just include them in 'core.py' instead.
+
+Jump into `lib/ansible/plugins/test <https://github.com/ansible/ansible/tree/devel/lib/ansible/plugins/test>`_ for details.
+
 .. _developing_callbacks:
 
-Callbacks
----------
+Callback Plugins
+----------------
 
-Callbacks are one of the more interesting plugin types.  Adding additional callback plugins to Ansible allows for adding new behaviors when responding to events.
+Callbacks are one of the more interesting plugin types. Adding additional callback plugins to Ansible allows for adding new behaviors tied to play events.
+The output you see from playbooks is controlled by callbacks, they can also be used to integrate with (send information to) other systems.
```

Comment:
The output you see from playbooks is controlled by callbacks. They can also be used to integrate with (send information to) other systems.